src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. Align the chained methods `.split`, `.map`, and `.filter` to improve readability as per the good example provided in the checklist.